test/e2e/app-dir/metadata-dynamic-routes/app/metadata-base/unset/opengraph-image2.tsx

```diff
@@ -0,0 +1,38 @@
+import { ImageResponse } from 'next/server'
+
+export const alt = 'Open Graph'
+
+export async function generateImageMetadata() {
+  return [
+    {
+      contentType: 'image/png',
+      size: { width: 1200, height: 600 },
+      id: 100,
+    },
+    {
+      contentType: 'image/png',
+      size: { width: 1200, height: 600 },
+      id: 101,
+    },
```

Comment:
Lets try a test when `id` is missing?

Reply:
Agreed as this will be easily forgotten.